indexTemplate.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const fs = require('fs');
+
+const updateIndex = (path, elementName) => {
+  // narrow down file directory to element .html files only
+  fs.readdir('./public', 'utf8', (err, files) => {
+    const elementFiles = files
+      .filter(file => {
+        return file.search('.html') !== -1;
+      })
+      .filter(file => {
+        return file !== 'index.html' && file !== '404.html';
+      });
+
+    let countElements = elementFiles.length;
+
+    // break up the HTML to insert new elements with ease
+    let indexHTML = `
+      <!DOCTYPE html>
+      <html lang="en">
+      <head>
+        <meta charset="UTF-8">
+        <title>The Elements</title>
+        <link rel="stylesheet" href="/css/styles.css">
+      </head>
+      <body>
+        <h1>The Elements</h1>
+        <h2>These are all of the known elements.</h2>
+        <h3>These are ${countElements} of them:</h3>
+        <ol>
+    `;
+
+    let indexHTMLClosed = `
+      </ol>
+      </body>
+      </html>
+    `;
+
+    // embed basic element information within li and anchor tags and append them to the HTML
+    const filesToHTML = elementFiles.map(file => {
```

Comment:
If you generate this string first, you can use just one string template literal by just injecting this single var into the single string template literal.